pkg/portal/regions_test.go

```diff
@@ -32,7 +33,7 @@ func TestRegionListPublic(t *testing.T) {
 		dbOpenShiftClusters: dbOpenShiftClusters,
 	}
 
-	os.Setenv("AZURE_ENVIRONMENT", azureclient.PublicCloud.Environment.Name)
+	os.Setenv("AZURE_ENVIRONMENT", strings.ToUpper(azureclient.PublicCloud.Environment.Name))
```

Comment:
You can remove this, if you use `strings.EqualFold()` in the comparison above.

Reply:
Removed